impacket/krb5/kerberosv5.py

```diff
@@ -75,6 +76,16 @@ def sendReceive(data, host, kdcHost):
         return r
 
     if krbError.getErrorCode() != constants.ErrorCodes.KDC_ERR_PREAUTH_REQUIRED.value:
+        if krbError.getErrorCode() == constants.ErrorCodes.KRB_AP_ERR_SKEW.value:
```

Comment:
We could move the code out of the `if krbError.getErrorCode()` statements to print the server time whenever the `sendReceive()` function manages to find it.

Reply:
After some thinking and testing, I think we shouldn't move the piece of code out of the if statements mentioned above. It would result in the server time being printed multiple times.

Reply:
Couldn't we print it when `KDC_ERR_PREAUTH_REQUIRED`?
```python
if krbError.getErrorCode() == constants.ErrorCodes.KDC_ERR_PREAUTH_REQUIRED.value:
    # Print server time
```